lib/nuca_backend_web/helpers/csv.ex

```diff
@@ -0,0 +1,63 @@
+defmodule NucaBackendWeb.CSV do
+  @base_datatypes [:string, :integer, :float]
+  import Plug.Conn, only: [put_resp_content_type: 2, send_resp: 3]
+
+  def to_csv(data, fields, row_parser) do
+    field_names =
+      fields
+      |> Enum.map(&elem(&1, 0))
+      |> Enum.join("; ")
```

Comment:
It would be interesting to consider the trade-offs between using semicolons as delimiters and using commas with each value wrapped with double-quotes e.g. `"(46.76042146260525, 23.599390521897618)"` for the coordinates.

Reply:
https://stackoverflow.com/a/10141043